src/accountant/core.cljs

```diff
@@ -111,14 +114,15 @@
   new page here.
 
   path-exists?: a fn of one argument, a path. Return truthy if this path is handled by the SPA"
-  [{:keys [nav-handler path-exists?]}]
+  [{:keys [nav-handler path-exists? reload-same-path?]}]
   (.setUseFragment history false)
   (.setPathPrefix history "")
   (.setEnabled history true)
   (set! accountant.core/nav-handler nav-handler)
   (set! accountant.core/path-exists? path-exists?)
+  (set! accountant.core/reload-same-path? path-exists?)
```

Comment:
I also don't think we need this, and if we do I don't understand why `path-exists?` would be the argument to it.